distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-ingest-plugins.adoc

```diff
@@ -0,0 +1,15 @@
+:title: Attributes Added by Pre-Ingest Plugins
+:type: dataManagement
+:status: published
+:parent: Automatically Added Metacard Attributes
+:summary: How pre-ingest plugins add attributes to metacards.
+:order: 05
+
+=== {title}
+
+The <<_pre_ingest_plugins,pre-ingest plugins>> are responsible for setting attribute fields on resulting metacards after the document has been transformed.
+
+* The <<_expiration_date_pre_ingest_plugin,expiration date pre-ingest plugin>> adds or updates expiration dates which can be used later for archiving old data.
+* The <<_geocoder_plugin,geocoder plugin>> is responsible for populating the Metacard’s `Location.COUNTRY_CODE` attribute if the Metacard has an associated location. If the metacard’s country code is already populated, the plugin will not override it.
```

Comment:
Can we link `Location.COUNTRY_CODE`? I think @ricklarsen added anchors for all the attrs

Reply:
The concern I have with that is that we don't have links on any of the other `Location.COUNTRY_CODE`s in the documentation . And I am a *sucker* for consistency.

However if this is really something we want to start doing, I would be perfectly happy starting that movement in this PR!